interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1083,6 +1083,7 @@ static void stringifyPreprocSetting(PreprocessorOptions& PPOpts,
       auto& HS = CI->getHeaderSearchOpts();
       addPrebuiltModulePaths(HS, getPathsFromEnv(getenv("LD_LIBRARY_PATH")));
       addPrebuiltModulePaths(HS, getPathsFromEnv(getenv("DYLD_LIBRARY_PATH")));
+      HS.AddPrebuiltModulePath(".");
```

Comment:
Did you mean cwd? What is cwg?

Reply:
pwd?

Reply:
What I mean is that there is a typo in the commit message.
Maybe it's better to change `cwg` to `current working directory`.

Reply:
Is `"."` converted into an absolute path the first time you "evaluate" it, or every time? I.e. what happens if you `cd` in between?

(IIRC believe it's converted into a full path the first time it's "evaluated". Is that the desired behavior?)

Reply:
It is not clear what's the desired behavior. If we want this to be dependent on changing the current dir.